compose/cli/main.py

```diff
@@ -863,6 +871,22 @@ def up(self, options):
             if cascade_stop:
                 print("Aborting on container exit...")
                 self.project.stop(service_names=service_names, timeout=timeout)
+                if forward_exitval:
+                    def is_us(container):
+                        return container.name_without_project == forward_exitval
+                    candidates = filter(is_us, all_containers)
```

Comment:
I actually tried that and ran into a scoping issue, but that was probably my own 2am incompetence -- I'll go commune with the documentation and try again.